### PR TITLE
Adds mappings for `ChannelAssetsDeletedByModerator` & `VideoAssetsDeletedByModerator` events

### PR DESCRIPTION
**Pull request by zeeshanakram3**
address #194

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1204404358291458/1205628042864801) by [Unito](https://www.unito.io)

### CODE REVIEW

File: src/mappings/content/video.ts [hunk 1]
```diff
@@ -169,6 +186,28 @@ export async function processVideoDeletedByModeratorEvent({
...
+
+  overlay.getRepository(Event).new({
+    ...genericEventFields(overlay, block, indexInBlock, extrinsicHash),
+    data: new VideoAssetsDeletedByModeratorEventData({
+      video: video.id,
+      assetIds,
+      deletedBy: parseContentActor(deletedBy),
+      rationale: rationale.toString(),
+    }),
+  })
```
Comment (ignazio-bovo) [inline src/mappings/content/video.ts:208]: Could you please check whether is it suitable to also remove the related `StorageDataObject` rows starting from `assetIds`. If you go through the `processVideoDeletion` I believe data objects are not deleted but in the rutnime they are.
So maybe the same considerations apply also there

File: src/mappings/content/channel.ts [hunk 2]
```diff
@@ -154,6 +156,28 @@ export async function processChannelDeletedByModeratorEvent({
...
+  block,
+  indexInBlock,
+  extrinsicHash,
+  overlay,
+  event: {
+    asV1000: [deletedBy, channelId, assetIds, rationale],
+  },
+}: EventHandlerContext<'Content.ChannelAssetsDeletedByModerator'>): Promise<void> {
+  const channel = await overlay.getRepository(Channel).getByIdOrFail(channelId.toString())
+
```
Comment (ignazio-bovo) [inline src/mappings/content/channel.ts:169]: Maybe also here you should deleted relevant `StorageDataObject`s.

**Review by ignazio-bovo — CHANGES_REQUESTED** [review]
My comments basically asks to evaluate whether we should remove related `StorageDataObjects`

**Comment by zeeshanakram3** [comment]
> My comments basically ask to evaluate whether we should remove related `StorageDataObjects`

Whenever the `ChannelAssetsDeletedByModerator` or `VideoAssetsDeletedByModerator` events are emitted by the runtime, the `DataObjectsDeleted` event is also emitted (this is due to the call made to Storage modules [delete_data_objects](https://github.com/Joystream/joystream/blob/a03082ebaf39e8e2006b0892c087a8b64f7abcbd/runtime-modules/content/src/lib.rs#L958) function from the extrisnsic).

So eventually the assets will be removed from Orion db in the [processDataObjectsDeletedEvent](https://github.com/zeeshanakram3/orion/blob/d9f370614559cd56b8d1cf36af770ffbf33e374b/src/mappings/storage/index.ts#L332) mapping, even before `*AssetsDeletedByModerator`  mappings are processed

**Pull request merged**